<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <item row="2" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>97</width>
         <height>11</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="6" rowspan="2">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Graph</string>
         </property>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>20</y>
            <width>81</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Distance X</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QSpinBox" name="sB_movex">
          <property name="geometry">
           <rect>
            <x>108</x>
            <y>30</y>
            <width>81</width>
            <height>26</height>
           </rect>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>3</x>
            <y>60</y>
            <width>111</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Draw Time (ms)</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="sB_update">
          <property name="geometry">
           <rect>
            <x>108</x>
            <y>60</y>
            <width>81</width>
            <height>26</height>
           </rect>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton">
          <property name="geometry">
           <rect>
            <x>3</x>
            <y>90</y>
            <width>101</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>moyenne /</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="sB_nbMoyenne">
          <property name="geometry">
           <rect>
            <x>108</x>
            <y>90</y>
            <width>81</width>
            <height>26</height>
           </rect>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2500000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>250</number>
          </property>
         </widget>
         <widget class="QCheckBox" name="cBscroll">
          <property name="geometry">
           <rect>
            <x>4</x>
            <y>120</y>
            <width>111</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>scroll graph</string>
          </property>
         </widget>
         <widget class="QPushButton" name="b_stop">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>122</y>
            <width>80</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>STOP</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>190</y>
            <width>191</width>
            <height>101</height>
           </rect>
          </property>
          <property name="title">
           <string>Consigne</string>
          </property>
          <widget class="QPushButton" name="send">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>30</y>
             <width>80</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>send</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>69</width>
             <height>26</height>
            </rect>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>42.000000000000000</double>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>71</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>show</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2" rowspan="2">
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="1" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Data :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdN_Data">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Moyenne :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdN_Moyenne">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0" rowspan="2" colspan="2">
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Serial Engine</string>
         </property>
         <widget class="QSpinBox" name="sB_Rand">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>26</y>
            <width>71</width>
            <height>26</height>
           </rect>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>26</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Debug Rand</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>56</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Read Speed</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="sB_speed">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>56</y>
            <width>71</width>
            <height>26</height>
           </rect>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
         <widget class="QPushButton" name="b_serClose">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>320</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>close</string>
          </property>
         </widget>
         <widget class="QPushButton" name="b_open">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>320</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>open</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>90</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Serial Name :</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="tE_Name">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>90</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;COM4&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>120</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>BaudRate</string>
          </property>
         </widget>
         <widget class="QComboBox" name="cB_DataBits">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>150</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>Data8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Data5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Data6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Data7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UnknownDataBits</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>154</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>DataBits</string>
          </property>
         </widget>
         <widget class="QComboBox" name="cB_BaudRate">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>120</y>
            <width>141</width>
            <height>25</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1200</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>190</y>
            <width>51</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Parity</string>
          </property>
         </widget>
         <widget class="QComboBox" name="cB_Parity">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>190</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>NoParity</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UnknownParity (obsolete)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EvenParity</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OddParity</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SpaceParity</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MarkParity</string>
           </property>
          </item>
         </widget>
         <widget class="QComboBox" name="cB_StopBits">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>230</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>UnknownStopBits (obsolete)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OneStop</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TwoStop</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OneAndHalfStop (windows only)</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>6</x>
            <y>230</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>StopBits</string>
          </property>
         </widget>
         <widget class="QComboBox" name="cB_FlowControl">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>270</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>NoFlowControl</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HardwareControl</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SoftwareControl</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>260</y>
            <width>51</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>Flow Control</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>7</x>
            <y>76</y>
            <width>201</width>
            <height>20</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="3">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>246</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="6">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>201</width>
         <height>9</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
